src/tokens.scss

```
@@ -0,0 +1,8 @@
+:host {
+  --ds-auro-banner-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
+  --ds-auro-banner-disclaimer-text-color: var(--ds-color-text-secondary-default, $ds-color-text-secondary-default);
+  --ds-auro-banner-title-text-color: var(--ds-color-text-emphasis-default, $ds-color-text-emphasis-default);
+  --ds-auro-banner-icon-background-color: var(--ds-color-icon-tertiary-default, $ds-color-icon-tertiary-default);
+  --ds-auro-banner-wrapper-background-color: var(--ds-color-brand-midnight-400, $ds-color-brand-midnight-400);
+  --ds-auro-banner-wrapper-border-color: var(--ds-color-border-primary-default, $ds-color-border-primary-default);
```

```suggestion
// Import Auro tokens
@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";

:host {
  --ds-auro-banner-text-color: var(--ds-color-text-primary-default, #{$ds-color-text-primary-default});
  --ds-auro-banner-disclaimer-text-color: var(--ds-color-text-secondary-default, #{$ds-color-text-secondary-default});
  --ds-auro-banner-title-text-color: var(--ds-color-text-emphasis-default, #{$ds-color-text-emphasis-default});
  --ds-auro-banner-icon-background-color: var(--ds-color-icon-tertiary-default, #{$ds-color-icon-tertiary-default});
  --ds-auro-banner-wrapper-background-color: var(--ds-color-brand-midnight-400, #{$ds-color-brand-midnight-400});
  --ds-auro-banner-wrapper-border-color: var(--ds-color-border-primary-default, #{$ds-color-border-primary-default});
```

Also, `$ds-color-icon-tertiary-default` is ** not** a token?